src/test/java/FileUploadTest.java

```diff
@@ -0,0 +1,23 @@
+import org.openqa.selenium.By;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FileUploadTest extends BaseTest {
+    @BeforeClass
+    public void openPage() {
+        driver.get("https://the-internet.herokuapp.com/upload");
+    }
+
+    @Test
+    public void testUpload() {
+        String absolutePath = (System.getProperty("user.dir") + "/src/test/resources/testForUpload.docx")
+                .replaceAll("\\\\", "/");
```

Comment:
what for?

Reply:
(System.getProperty("user.dir") ) returned me part of path with left slash '\' and test failed